Update ADK doc according to issue #1113 - 2 - LlmAgent set_default_model (#1116)
google/adk-docs: docs/agents/llm-agents.md

            .build();
    ```

### Configuring a Default Model

> You can set a system-wide default model for all `LlmAgent` instances using the `set_default_model` class method. If you do not specify a model when creating an agent, it will use this default. This is useful for avoiding redundant model specifications and for easily changing the model for all agents at once.
> 
> The built-in default model is `gemini-2.5-flash`.

Reviewer: We can drop the pinned version here, it's already stale since this PR was created (main is on gemini-3.5-flash, not 2.5) and the exact current value is kept up to date in the auto-generated API reference, so we don't need to hardcode it in prose.

```suggestion
You can set a system-wide default model for all `LlmAgent` instances using the `set_default_model` class method. If you do not specify a model when creating an agent, it falls back to ADK's built-in default model. This is useful for avoiding redundant model specifications and for easily changing the model for all agents at once.
```

=== "Python"

    ```python
    from google.adk.agents import LlmAgent

    # Set a new default model for all agents

>     LlmAgent.set_default_model("gemini-pro")

Reviewer: Use `gemini-flash-latest` here to match the alias we use throughout the docs.

>     # This agent will now use "gemini-pro" by default

Reviewer: Update this comment to `gemini-flash-latest` to match.

    agent_with_default_model = LlmAgent(
        name="default_model_agent",
        instruction="You are a helpful assistant."
    )

    # You can still override the default for specific agents
    specific_agent = LlmAgent(
        name="specific_model_agent",

>         model="gemini-2.5-flash",

Reviewer: Change to `gemini-pro-latest` so the override is meaningful (shows a different model than the default) and avoids a pinned version.
